.github/workflows/check_codestyle.yml

```diff
@@ -2,13 +2,19 @@ name: 'Check: Codestyle'
 
 on:
   pull_request:
+  workflow_dispatch:
 
 jobs:
   codestyle:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
     steps:
     - name: Clone repository
       uses: actions/checkout@v4
+
+    - name: Fetch Muse CI scripts
+      run: |
+        cmake -P ./buildscripts/ci/checkcodestyle/ci_fetch.cmake
+
     - name: Test coding style
       run: |
-        bash ./buildscripts/ci/linux/checkcodestyle.sh
+        cmake -P ./buildscripts/ci/checkcodestyle/_deps/checkcodestyle.cmake ./src/
```

Comment:
Seems youi forgot to add this file?

Reply:
In the previous step we download this file from another repository (framework repository), so there is no problem
The idea of ​​this PR is to reuse scripts from the framework

Reply:
OK, that's what I gathered myself meanwhile too ;-)